### PR TITLE
chore: use NPM workspaces

### PR DESCRIPTION
**Pull request by frbuceta**

### CODE REVIEW

**Comment by frbuceta** [comment]
@shubhamp-sf Do you know what might be going on here?

**Comment by shubhamp-sf** [comment]
@frbuceta I see the code changes removed package json and lerna bootstrap step. I tried it locally but it didn't install packages for me. So sequelize and typeorm are looking for that package and not finding it.

File: package.json [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
   "copyright.owner": "IBM Corp. and LoopBack contributors",
   "repository": "github:loopbackio/loopback-next",
-  "engineStrict": true,
-  "engines": {
-    "node": "16 || 18 || 20",
-    "npm": ">=7"
-  },
   "scripts": {
     "prepare": "husky install",
-    "postinstall": "lerna bootstrap && npm run -s update-ts-project-refs",
```
Comment (shubhamp-sf) [inline package.json:17]: This might be the cause because lerna bootstrap is what was responsible for installing npm packages in our packages.

**Comment by shubhamp-sf** [comment]
Btw why are we moving onto NPM workspaces here? Is there any discussion thread I can go through?

**Comment by frbuceta** [comment]
Also, the project installs the dependencies super fast and that makes the tests in github run much faster.

**Comment by frbuceta** [comment]
> Oh okay, then IMO the only culprit can be `require()` calls in both typeorm and sequelize for the reason that they require it conditionally It is unable to check them in correct `node_modules`.

Yes, that's right. The strange thing is that it only fails in GitHub Actions.

**Comment by frbuceta** [comment]
@raymondfeng Can you help me understand what's going on with webpack, with the web compilation?

**Pull request merged**